app/resources/results.html.template

```diff
@@ -82,214 +82,233 @@
+                {% endif %}
+                {% if result.home_city %}
+                  {{result.home_city}}&nbsp;&nbsp;
+                {% endif %}
+                {% if result.home_state %}
+                  {{result.home_state}}&nbsp;&nbsp;
+                {% endif %}
+                {% if result.home_postal_code %}
+                  {{result.home_postal_code}}&nbsp;&nbsp;
+                {% endif %}
+              </span>
+            </div>
+            {% endif %}
+            <div>
+              <span class='resultDataLabel'>
+                {% trans "Status" %}:
+              </span>
+              <span class='resultDataPersonFound resultDataValue'>
+                {{ result.latest_note_status }}
+              </span>
+            </div>
+            {% if result.provider_name %}
+            <div>
+              <span class='resultDataLabel'>
+                {% trans "Provided by:" %}
+              </span>
+              <span class='resultDataValue'>
+                {{result.provider_name}}
+              </span>
+            </div>
+            {% endif %}
+            {% if env.show_record_ids_in_results %}
+              <span class="tracking">
+                ID: {{result.record_id}}
+              </span>
+            {% endif %}
           </div>
-          {% endif %}
-          {% if env.show_record_ids_in_results %}
-            <span class="tracking">
-              ID: {{result.record_id}}
-            </span>
-          {% endif %}
+          <br clear='all'>
         </div>
-        <br clear='all'>
-      </div>
-    {% endfor %}
-    </div>
-    {% if env.enable_dup_mode %}
-      <div id='dup_form' style='display:none'>
-        {% trans "Select up to 3 records to mark as duplicate:" %}
-        <form action="{{env.repo_path}}/multiview" method="get" onsubmit="mark_dup()">
-          {{env.hidden_input_tags_for_preserved_query_params|safe}}
-          <input type='hidden' name='id1' id='id1' value=''>
-          <input type='hidden' name='id2' id='id2' value=''>
-          <input type='hidden' name='id3' id='id3' value=''>
-          <input id='dup_go' disabled='disabled' type='submit' value='{% trans "Mark the selected records as duplicate" %}'>
-          {% blocktrans with "<span id='dup_count'>0</span>"|safe as dup_count %}
-            {{dup_count}} records selected
-          {% endblocktrans %}
-        </form>
+      {% endfor %}
       </div>
+      {% if env.enable_dup_mode %}
+        <div id='dup_form' style='display:none'>
+          {% trans "Select up to 3 records to mark as duplicate:" %}
+          <form action="{{env.repo_path}}/multiview" method="get" onsubmit="mark_dup()">
+            {{env.hidden_input_tags_for_preserved_query_params|safe}}
+            <input type='hidden' name='id1' id='id1' value=''>
+            <input type='hidden' name='id2' id='id2' value=''>
+            <input type='hidden' name='id3' id='id3' value=''>
+            <div class='button-cancel'>
```

Comment:
button-cancel makes the button red. I thought it would be appropriate for "Mark records as duplicate" to be red as it indicates there's an error in the data.

Would you prefer me to change it to blue? Or to chose a different name for the CSS class?